docs/recipes/typescript.md

```diff
@@ -50,6 +50,23 @@ test(async (t) => {
 });
 ```
 
+## Working with [macros](https://github.com/avajs/ava#test-macros)
+
+In order to be able to assign the `title` property to a macro:
+
+```ts
+import test, { AssertContext, Macro } from 'ava';
+
+const macro: Macro<AssertContext> = (t: AssertContext, input, expected) => {
```

Comment:
Looking at this in VS Code the type of `t` is inferred. Given how often you'd end up typing that parameter perhaps the example shouldn't explicitly type it.

Reply:
You're right, because:
https://github.com/avajs/ava/blob/64b7755d97cba07ed62d797c6ce2f85ffa11a01e/types/base.d.ts#L118